README.md

```diff
@@ -65,9 +65,10 @@ e2e
 |     tests as well as methods required by the tests to run
 ```
 
-## Guide for writing the tests
+## Guide for writing tests
 
-When writing a new test case, you need to create a new spec in `./e2e/tests`
+When writing a new test case, create a spec in `./e2e/tests`
 
-## Github Action integration
+## GitHub Actions integration
 The e2e.yml workflow is made up of one job that is triggered by PRs, and on a push.
+The env.yml workflow is made up of one job that supports selection of a preferred environment at runtime.
```

Comment:
Lets adopt the docs here following [this comment template](https://github.com/ozone-his/ozone-e2e-pro/pull/60#pullrequestreview-1727373150).

Reply:
→ [6fe1183](https://github.com/mekomsolutions/ozone-e2e-kh/pull/24/commits/6fe11838ef9ca0dac2b955993dc475fe48ebb593)